irods_consortium_continuous_integration_build_hook.py

```diff
@@ -18,6 +18,7 @@ def install_building_dependencies(externals_directory):
     externals_list = [
         'irods-externals-avro1.11.0-3',
         'irods-externals-boost1.81.0-1',
+        'irods-externals-clang-runtime13.0.1-0',
```

Comment:
This branch doesn't need `clang-runtime`, as it's build against libstdc++

Reply:
Good call. Created https://github.com/irods/irods_rule_engine_plugin_audit_amqp/issues/148 for this.